test.py

```diff
@@ -1,8 +1,34 @@
 import pandas as pd 
 import matplotlib.pyplot as plt
+import numpy as np 
 
 filename = "data/trigs001a.001"
 data = pd.read_csv(filename, skiprows=188, sep='\t', index_col='time')
-data[["A-Ph1", "A-Ph2", "A-Ph3", "A-Ph4", "A-Ph5"]].plot()
+
+# data[["A-Ph1", "A-Ph2", "A-Ph3", "A-Ph4", "A-Ph5"]].plot()
+# plt.show()
+
+detectors = ['A', 'B', 'C', 'D']
+fig, ax = plt.subplots()
+
+count_chan = 0
+sumchan = pd.Series(np.arange(len(data)))
+
+# Load distance file
+# Find short channels
+
+for detector in detectors:
```

Comment:
practicing looping through channels, normalizing them in z scores (x - mean (X)) / std(X),  averaging them, and plotting the average,